Maple2.File.Parser/Xml/Table/GachaInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace Maple2.File.Parser.Xml.Table;
+
+// ./data/xml/table/gacha_info.xml
+[XmlRoot("ms2")]
+public partial class GachaInfoRoot {
+    [XmlElement] public List<GachaInfo> randomBox;
+}
+
+public partial class GachaInfo {
+    [XmlAttribute] public int randomBoxID;
+    [XmlAttribute] public byte randomBoxGroup;
+    [XmlAttribute] public byte randomBoxSeasonKR;
```

Comment:
idk wha the xml looks like but are these fields always present? if not maybe you should make them nullable so you can check if they are set.

Reply:
randomBoxID and randomBoxGroup are always present. randomBoxSeasonKR is as well but value is sometimes not defined. I did use the sanitizer for this.